index.html

```diff
@@ -1137,6 +1138,10 @@ <h2>Generate a network error report</h2>
             </dd>
           </dl>
         </li>
+        <li>
+          If <var>additional body</var> is present, set each properties of

```

Comment:
This isn't precise enough. I think you intend to run approximately the algorithm from [Object.assign](https://tc39.github.io/ecma262/#sec-object.assign), which nails down whether non-enumerable properties are included, what happens if the argument is a proxy, etc.

Using a WebIDL dictionary for additional body would be easier to specify precisely, and would satisfy @igrigorik's concern in https://github.com/w3c/network-error-logging/issues/99#issuecomment-453839582.

Reply:
Uploaded https://github.com/w3c/network-error-logging/pull/100/commits/72f2bd6a0b2a76c200ef3797a08f0003c6698d58 which introduces AdditionalReportBody dictionary.
How about this?